### PR TITLE
Enable Onnxruntime Telemetry by Default for 1.3

### PR DESCRIPTION
**Pull request by zhangxiang1993**
This PR is to enable onnxruntime telemetry collection by default by setting bool WindowsTelemetry::enabled_ = true

### CODE REVIEW

File: winml/lib/Api.Ort/OnnxruntimeEngine.cpp [hunk 1]
```diff
@@ -1259,7 +1259,6 @@ const WinmlAdapterApi* OnnxruntimeEngineFactory::UseWinmlAdapterApi() {
 HRESULT OnnxruntimeEngineFactory::GetOrtEnvironment(OrtEnv** ort_env) {
   RETURN_IF_FAILED(EnsureEnvironment());
   RETURN_IF_FAILED(environment_->GetOrtEnvironment(ort_env));
-  RETURN_HR_IF_NOT_OK_MSG(ort_api_->EnableTelemetryEvents(*ort_env), ort_api_);
```
Comment (tiagoshibata) [inline winml/lib/Api.Ort/OnnxruntimeEngine.cpp]: Why is this removed?

Comment (martinb35) [reply winml/lib/Api.Ort/OnnxruntimeEngine.cpp]: because it's on by default now, so winml doesn't need to turn it on.
